File: MODUL 2/Soal 10/PRAK110-2210817310008-Muh_Haekal_Barera.py
import math
alas = 5
tinggi = 12
SB = math.sqrt((alas*alas)+(tinggi*tinggi))
keliling = alas + tinggi + SB
luas = (alas*tinggi)/2
print("Diketahui :")
print("Alas = {} cm".format(alas))
print("Tinggi = {} cm\n".format(tinggi))
print("Jawab :")
print("Sisi A = {} cm".format(tinggi))
print("Sisi B = {} cm".format(round(SB)))
print("Sisi C = {} cm".format(alas))
print("Keliling = {} cm".format(round(keliling)))
print("Luas = {} cm".format(round(luas)))

File: MODUL 2/Soal 10/PRAK110-2210817310008-Muh_Haekal_Barera.c
#include <stdio.h>
#include <math.h>

int main()
{
    int alas, tinggi, sA, sB, sC, keliling, Luas, axt;
    alas = 5;
    tinggi = 12;
    sA = alas;
    sC = tinggi;
    axt = (alas * alas) + (tinggi * tinggi);
    sB = sqrt(axt);
    keliling = sA + sB + sC;
    Luas = (sA * sC) / 2;
    printf("Diketahui : \n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n\n", tinggi);
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", sA);
    printf("Sisi B = %d cm\n", sB);
    printf("Sisi C = %d cm\n", sC);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm\n", Luas);
    return 0;
}